requirements.txt

```diff
@@ -45,3 +45,7 @@ python-glanceclient
 python-keystoneclient
 python-novaclient
 python-blazarclient
+
+celery
+redis
```

Comment:
Why do we need this?

Reply:
The space? There seems to be some kind of organizing structure in the requirements text. I moved celery, redis, and kombu up with the list of non-openstack and non-django related requirements.